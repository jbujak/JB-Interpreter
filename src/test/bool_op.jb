int main() {
	print("neg");
	print(bool_to_string(!false));
	print(bool_to_string(!true));
	print("");

	print("and");
	print(bool_to_string(false && false));
	print(bool_to_string(false && true));
	print(bool_to_string(true  && false));
	print(bool_to_string(true  && true));
	print("");

	print("or");
	print(bool_to_string(false || false));
	print(bool_to_string(false || true));
	print(bool_to_string(true  || false));
	print(bool_to_string(true  || true));
}
