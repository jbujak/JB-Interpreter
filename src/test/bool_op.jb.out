neg
true
false

and
false
false
false
true

or
false
true
true
true
